src/Stepper/Step.tsx

```diff
@@ -0,0 +1,111 @@
+import * as classnames from "classnames";
+import * as React from "react";
+import * as PropTypes from "prop-types";
+
+import "./Step.less";
+import CheckMark from "./CheckMark";
+import {FlexBox} from "../flex";
+import Exclamation from "./Exclamation";
+
+const propTypes = {
+  className: PropTypes.string,
+  title: PropTypes.string,
+  description: PropTypes.oneOfType([PropTypes.node]),
```

Comment:
`description: PropTypes.node,`

"Drop the `PropTypes.oneOfType`. It's cleaner." - Sean Parker